src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -67,9 +69,13 @@ public static List<Person> getTypicalPersons() {
         return new ArrayList<>(Arrays.asList(
                 ALICE,
                 BENSON,
-                CARL,
                 BOB,
-                CARL_GOH
+                CARL,
+                CARL_GOH,
+                DON,
+                JAMES,
```

Comment:
Can we please follow naming conventions... keep it to alphabetical... why J all of a sudden?

Reply:
The James was not implemented by me.